Quantum-Control-Applications-QuAM/Superconducting/configuration/make_pulses.py

```diff
@@ -54,7 +54,7 @@ def add_default_transmon_pulses(transmon):
         digital_marker="ON",
     )
     transmon.xy.operations["-y90_DragGaussian"] = pulses.DragPulse(
-        amplitude=-0.1 / 2,
```

Comment:
For negative pulses I would suggest to keep positive amplitudes and use negative `axis_angle`. This way you can also keep the amplitude in just one pulse